pages/express-relay/integrate-as-searcher.mdx

```diff
@@ -0,0 +1,258 @@
+import { Callout, Tabs, Steps } from "nextra/components";
+
+# How to Integrate Express Relay as a Searcher
+
+Express Relay gives one place stop for on-chain opportunities. Searchers can access opportunities from various DeFi protocols via a simple and unified interface.
+
+Searchers **bid** on opportunities exposed by the Express Relay server.
+The server exposes different endpoints for interaction, which can be used directly via HTTP, WebSocket, or one of the SDKs for convenience.
+
+Searchers can integrate with the Express Relay server in three steps:
+
+1. Subscribe to new opportunities
+2. Evaluate the opportunity and construct the bid
+3. Submit the bid to the auction server
+
+<Steps>
+
+### Subscribing to new opportunities
```

Comment:
It's in the title, maybe words should be capitalized like: "Subscribing to New Opportunities"

Reply:
Opportunites.